src/index.js

```diff
@@ -1,18 +1,35 @@
 import express from 'express';
+import passport from 'passport';
+import session from 'express-session';
+import cors from 'cors';
+import logger from 'morgan';
 import apiRouter from './api/routes/index';
 import docsRouter from './api/routes/docs';
 import homeRouter from './api/routes/home';
-import register from './middlewares/register.app';
 import env from './configs/environments';
 
+import passportConfig from './middlewares/passport';
+import './helpers/cloudnary';
+
 const app = express();
+app.use(cors());
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(logger('dev'));
+
+app.use(express.static(`${__dirname}/public`));
+app.use(passport.initialize());
+passportConfig(passport);
 
-// Register middleware
-register(app);
+app.use(session({
+  secret: env.secret,
```

Comment:
do we need session? if yes, why?

Reply:
We need it for set response cookies and read from the response.
you can read more about on https://www.npmjs.com/package/express-session